src/schedules/service/SchedulesService.ts

```diff
@@ -0,0 +1,34 @@
+import RequestClientWrapper from '../../RequestClientWrapper';
+import Version from '../../model/Version';
+import Resource from '../../model/Resource';
+import { AxiosResponse } from 'axios';
+import GenerateApiError from '../../utils/GenerateApiError';
+import * as Q from 'q';
+import Schedule from '../model/Schedule';
+
+const EXPECTED_GET_STATUS = 200;
+export default class SchedulesService {
+  constructor(
+      private readonly client: RequestClientWrapper
+    ) { }
+  public get(businessId: string): Promise<Schedule[]> {
```

Comment:
We actually need a method for getting a single schedule by schedule ID, not business ID.  I just updated the task to be more clear about that.
However you can leave in the `get(businessId: string)` method as I'm sure we will eventually need it for something else.